.github/workflows/mend.yml

```diff
@@ -0,0 +1,34 @@
+name: Mend
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - design/**
+      - deploy**
+      - docs/**
+      - examples/**
+
+concurrency:
+  group: ${{ github.ref_name }}-mend
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  scan:
+    name: Mend
+    runs-on: ubuntu-22.04
+    if: ${{ github.event.repository.fork == false }}
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - name: Download agent
+        run: curl -LJO https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
```

Comment:
I can add that.

Reply:
Since we don't pass `jarsigner -verify wss-unified-agent.jar` any additional information (like a certificate or a hash sum), I wonder how it can verify?

I was thinking of using a sum from `https://github.com/whitesource/unified-agent-distribution/releases/download/v23.5.2.1/wss-unified-agent.jar.sha256` , storing the sum string in the mend.yaml.  But for that, we need to download specific release of white source, not latest 🤔

Reply:
Check out the `JAR File Verification` section [here](https://docs.oracle.com/en/java/javase/17/docs/specs/man/jarsigner.html).

Reply:
thanks. but should we also import the public key? like here https://docs.oracle.com/javase/tutorial/security/toolfilex/rstep1.html ?

Reply:
For one, where would we get that key? For two, that process requires manual verification that the fingerprints match what are expected.

Reply:
> For one, where would we get that key? For two, that process requires manual verification that the fingerprints match what are expected.

kk.

looks like it is signed by a chain leading to a root cert https://github.com/whitesource/unified-agent-distribution/issues/46